src/native/ffi.ts

```diff
@@ -165,11 +167,18 @@ const symbols = {
   },
 } as const;
 
-export default Deno.dlopen(
-  new URL(`../../native/build/${Deno.build.os === "windows" ? "" : "lib"}netsaur.${Deno.build.os === "linux" ? "so" : Deno.build.os === "darwin" ? "dylib" : "dll"}`, import.meta.url),
-  symbols,
-)
-  .symbols;
+const opts: FetchOptions = {
+  name: "netsaur",
+  url: "https://github.com/denosaurs/netsaur/releases/download/0.1.4/",
+  prefixes: {
```

Comment:
I think you want to add an `suffixes` property to properly suffix aarch64

Reply:
support for aarch64 isn't quite there yet